kurtosis-devnet/tests/fees/fees_test.go

```diff
@@ -141,12 +141,12 @@ func feesTestScenario(
 
 		// Get the L2 client
 		l2Chain := llsys.L2s()[chainIdx]
-		l2Client, err := l2Chain.Client()
+		l2Client, err := l2Chain.LowLevelClient()
```

Comment:
I like this change, but syntactically it would be preferable if clients weren't tightly bound to a chain object, but instead were bound to a node object (where there is many-to-one relationship between nodes and chains).

Reply:
Agree. Current chain interface only has ability for a clients rooted in single RPC, but if we support more RPCs, each node should have clients not chains. Tracked at https://github.com/ethereum-optimism/optimism/issues/14811.

Reply:
The short-term redundancy between the Node interface and the sources.Client struct seems ok for the sake of keeping PRs small.